lib/common/util.ml

```diff
@@ -45,7 +45,7 @@ end
 module Net = struct
   let create_socket port =
     let open Lwt_unix in
-    let ssock = socket PF_INET SOCK_DGRAM 0 in
+    let ssock = socket ~cloexec:true PF_INET SOCK_DGRAM 0 in
```

Comment:
What does this do?

Reply:
Comment got lost on different re-work:
https://github.com/marigold-dev/pollinate/pull/19#discussion_r838552390